Run dependabot for go weekly, all updates at once

dfarrell07 opened the pull request:
To minimize the code review work required to merge go update PRs, cause
dependabot to give us all the updates at the same time, once a week.
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our developer guide: https://submariner.io/development/
2. Ensure you have added the appropriate tests for your PR: https://submariner.io/development/code-review/#test-new-functionality
3. Read the code review guide to ease the review process: https://submariner.io/development/code-review/
4. If the PR is unfinished, mark it as a draft: https://submariner.io/development/code-review/#mark-work-in-progress-prs-as-drafts
5. If you are using CI to debug, use your private fork: https://submariner.io/development/code-review/#use-private-forks-for-debugging-prs-by-running-ci
6. Add labels to the PR as appropriate.

This template is based on the K8s/K8s template:

https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md
-->

skitt commented on .github/dependabot.yml:
```diff
@@ -25,14 +25,8 @@ updates:
     schedule:
       interval: daily
     open-pull-requests-limit: 10
-    ignore:
-      # Our own dependencies are handled during releases
-      - dependency-name: github.com/submariner-io/*
```
Does this actually work on devel commits? What we want is bumps to the latest commit on the devel branches; I wonder if this won’t cause dependabot to submit PRs updating to the latest tag, either on devel or on the last release branch once it exists (which will do the wrong thing).

dfarrell07 replied:
I'll remove that change for now, send a different PR with a different attempt.

dfarrell07 replied:
Try two: https://github.com/submariner-io/submariner/pull/2173

dfarrell07 committed:
Run dependabot for go weekly, all updates at once

To minimize the code review work required to merge go update PRs, cause
dependabot to give us all the updates at the same time, once a week.

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

dfarrell07 commented:
[E2E (1.22)](https://github.com/submariner-io/lighthouse/actions/runs/3622404239/jobs/6107072065#logs) timed out, trying again.
